content/en/docs/configuration/acme/http01/_index.md

```diff
@@ -121,3 +121,36 @@ The added labels and annotations will merge on top of the cert-manager defaults,
 overriding entries with the same key.
 
 No other fields of the `podTemplate` exist.
+
+### `ingressTemplate`
+
+It is possible to add labels and annotations to the solver ingress resources.
+These can be configured under the `metadata` field under `ingressTemplate`:
+
+```yaml
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
Can we make the version `v1alpha3`?

Reply:
All docs samples are currently v1alpha2, and given a subset of users will be unable to use v1alpha3 yet, I think it makes sense for us to hold these back for now (and update all manifests at once). Consistency is best IMO 🙂

Reply:
:+1: